common_types/src/datum.rs

```diff
@@ -620,6 +708,35 @@ impl Datum {
         }
     }
 
+    fn parse_datum_time_from_str(s: &str) -> Result<Datum> {
```

Comment:
Maybe we should add some tests for parsing and formating of `Datum::Date` and `Datum::Time`, and some corner cases should be included.

Reply:
done